### ISSUE TITLE
Merge/delete/move around files in directory

### ISSUE THREAD

**Issue by lzhaxi** [18nshinagawa/notes#71]
Should be a way to merge directories (I accidentally made a stats directory instead of Stats) and/or delete directories.

**Comment by lzhaxi** [comment]
Also I realized I put a different file by itself and not in Stats at all

**Comment by 18nshinagawa** [comment]
good idea, but this seems like a big project; i need completely reorganize the whole menu thing yet again, it seems
